helm/examples/azure/sa/create-sa.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
```

Comment:
as mentioned in a previous comment - might be okay to have defaults coming from a file,
but should really be able to specify configuration settings on the command line for these scripts.

Reply:
I am little concern about inconsistent inputs from users for create-sa.sh , create-secret.sh, delete.sh and list.sh Thought have one place for the configuration is better. I will may re-think it later.

Reply:
ok, but potential inconsistency from user error on command line, is no different from any other tool, e.g. the az command line itself, they must ensure aks name, resource group name, etc. are consistent.
But that is standard, and allows the standard scripting, without having to write settings to a external file, to be read by scripts.

I think ok for now, but worth adding a JIRA to revisit - and referencing/linking to this JIRA - so easily found.